hypershift-operator/controllers/nodepool/kubevirt/kubevirt.go

```diff
@@ -271,6 +280,21 @@ func virtualMachineTemplateBase(nodePool *hyperv1.NodePool, bootImage BootImage)
 		template.Spec.Template.Spec.NodeSelector = kvPlatform.NodeSelector
 	}
 
+	if kvPlatform.KubevirtHostDevices != nil && len(kvPlatform.KubevirtHostDevices) > 0 {
+		hostDevices := []kubevirtv1.HostDevice{}
+		for _, hostDevice := range kvPlatform.KubevirtHostDevices {
+			for i := 1; i <= hostDevice.Count; i++ {
+				kvHostDevice := kubevirtv1.HostDevice{
+					Name:       "gpu" + strconv.Itoa(i),
```

Comment:
In order to keep with the idea that this might be used for more generic devices in the future, maybe instead of calling this `gpu-` we should use `hostdevice-`

Reply:
done